bin/faucet/src/handlers.rs

```diff
@@ -23,14 +29,14 @@ struct FaucetRequest {
 #[derive(Serialize)]
 struct FaucetMetadataReponse {
     id: String,
-    asset_amount_options: Vec<u64>,
+    asset_amount_options: [u64; 3],
```

Comment:
why do we need the change to a fixed size array?

Reply:
This was one of the changes I left on my PR to `main`, but I think @tomyrd is going to roll it back into using the configuration field.

Reply:
Done.